package.json

```diff
@@ -41,7 +41,7 @@
     "@ngrx/effects": "6.1.0",
     "@ngrx/entity": "6.1.0",
     "@ngrx/router-store": "6.1.0",
-    "@ngrx/store": "6.1.0",
+    "@ngrx/store": "^8.6.0",
```

Comment:
any breaking changes?

Reply:
The changes suggested in https://github.com/bwsw/cloudstack-ui/issues/1357 require `@ngrx/store/testing` but it's missing in the 6th version.